gamemodes/terrortown/gamemode/client/cl_scoring.lua

```diff
@@ -285,23 +302,40 @@ function CLSCORE:SaveLog()
 	local logname = logdir .. "/ttt_events_" .. os.time() .. ".txt"
 	local log = "Trouble in Terrorist Town 2 - Round Events Log\n" .. string.rep("-", 50) .. "\n"
 
-	log = log .. string.format("%s | %-15s | %s\n", " TIME  ", "TYPE", "WHAT HAPPENED") .. string.rep("-", 50) .. "\n"
+	log = log
+		.. string.format("%s | %-15s | %s\n", " TIME  ", "TYPE", "WHAT HAPPENED")
+		.. string.rep("-", 50)
+		.. "\n"
 
 	for i = 1, #events do
 		local event = events[i]
 
-		if event.event.roundState ~= ROUND_ACTIVE then continue end
+		if event.event.roundState ~= ROUND_ACTIVE then
+			continue
+		end
 
 		local time = event:GetTime()
 		local minutes = math.floor(time / 60)
 		local seconds = math.floor(time % 60)
 
-		log = log .. string.format("[%02d:%02d] | %-15s | %s\n", minutes, seconds, event.type, event:Serialize() or "")
+		log = log
+			.. string.format(
+				"[%02d:%02d] | %-15s | %s\n",
+				minutes,
+				seconds,
+				event.type,
+				event:Serialize() or ""
+			)
```

Comment:
This should be

```lua
log = log .. string.format(
	"[%02d:%02d] | %-15s | %s\n",
	minutes,
	seconds,
	event.type,
	event:Serialize() or ""
)
```

Reply:
fwiw: in jetbrains IDEs, these formatting options are referred to as "chopping", as in, they only ever get split into multiline things if they exceed a breakpoint line length, and IMO most of the time this is the only thing that really matters: a line is too long, it should be split across multiple lines in a clear and consistent fashion.

Reply:
I get it, I also think this multiline version is much more readable. But I don't get why it isn't how I suggested it. This is my main gripe with it

Reply:
i thought i was replying to the thread and not a specific annotation, my bad, i meant as overall; some of the other things in the chain get split up seem excessive to do _always_, like splitting if (...) then across 3 lines but not splitting the condition itself after 80 characters across each logical operator or parenthetical group